src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -0,0 +1,169 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
+import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
+import com.google.cloud.language.v1.ClassificationCategory;
+import com.google.cloud.language.v1.ClassifyTextRequest;
+import com.google.cloud.language.v1.ClassifyTextResponse;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.EncodingType;
+import com.google.cloud.language.v1.Entity;
+import com.google.cloud.language.v1.EntityMention;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/*
+* This class creates objects that analyse text and 
+*/
+public final class TextAnalyser {
+  private final String message;
+
+  public TextAnalyser(String message) {
+     this.message = message.toLowerCase();
+  }
+
+  public float getSentimentScore() throws IOException {
+    try (LanguageServiceClient languageService = LanguageServiceClient.create()) {
+      Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+      Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+        
+      return sentiment.getScore();
+    }
+  }
+
+  public String getMood() throws IOException {
+    float score = getSentimentScore();
+    
+    if(score == 1) {
+      return "very happy";
+    }
```

Comment:
Since score variable is a float, you might need to avoid this comparison technique. For comparing floating point data types I would chose something similar to this: 
if(Math.abs(score - 1) < epsilon)
https://stackoverflow.com/questions/1088216/whats-wrong-with-using-to-compare-floats-in-java